tests/db/jobs/test_daemon.py

```diff
@@ -64,16 +77,23 @@ def test_start_stop(job_controller, single, daemon_manager) -> None:
     assert daemon_manager.check_status() == DaemonStatus.STOPPED
 
     assert daemon_manager.start(raise_on_error=True, single=True)
-    time.sleep(0.5)
+    if not single:
+        warn_msg = (
+            "Daemon is STOPPED, but the options single differ from the values used "
+            "to activate supervisor. The daemon will start with the initial configurations"
+        )
+        assert warn_msg in caplog.text
+
+    wait_daemon_started(daemon_manager)
     assert daemon_manager.shut_down(raise_on_error=True)
-    time.sleep(1)
+    _wait_daemon_shutdown(daemon_manager)
```

Comment:
Why not using the wait_daemon_shutdown fixture ?

Reply:
switched to the fixture